doc/Testing.md

```diff
@@ -8,6 +8,20 @@ To run tests on theia run:
 
 This will run all CI enabled tests.
 
+If you want to run all tests for a particular Theia extension, execute the following command from the root:
+
+`npx run test @theia/extension-name`
+
+If you would like to enable watch mode for the tests, add the following npm script to the `package.json` of the desired Theia extension.
+
+```json
+  "test:watch": "theiaext test:watch"
+```
+
+After editing the `package.json`, execute `yarn` in the Theia root, then you can run the tests in watch mode with:
```

Comment:
it is not required after changing `package.json`, calling yarn should be enough once after cloning the repo

Reply:
It was not the case, that's why I have [asked](https://github.com/theia-ide/theia/issues/924#issuecomment-348461744).

Reply:
I am sure that `theiaext` in binaries of all extensions after running yarn and adding a new npm scirpt does not change there theiext should be added as a binary or not. Maybe you did something like `git clean -dfx` and forget to run `yarn` again?

Reply:
Alright, I'll try it again before merging this PR.